code/controllers/subsystem/ticker.dm

```diff
@@ -217,11 +217,11 @@ SUBSYSTEM_DEF(ticker)
 			for(var/obj/effect/landmark/spacepod/random/R in L)
 				qdel(R)
 
-		to_chat(world, "<FONT color='blue'><B>Enjoy the game!</B></FONT>")
+		to_chat(world, "<FONT color='#6685f5'><B>Enjoy the game!</B></FONT>")
```

Comment:
Although saying that, couldnt you replace all these with a custom span and put that in the chat? The ones with ternaries dont matter. `pmsend` for example

Reply:
Good idea, done.